lib/atrium/institution.rb

```diff
@@ -8,7 +8,10 @@ class Institution
     attribute :name
     attribute :small_logo_url
     attribute :url
+    attribute :medium_logo_url
+    attribute :small_logo_url
```

Comment:
Looks like this wasn't rebase correctly. Both small and medium uris are duplicated here.